ampersand-form-view.js

```diff
@@ -31,6 +31,12 @@ module.exports = View.extend({
         }
     },
 
+    template: [
+        '<form>',
+            '<fieldset data-hook="field-container"></fieldset>',
```

Comment:
I used a [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), which from what I understand is the most appropriate field container element. Technically you could do without this line and put the `data-hook` in the `<form>` (or even leave the hook out because it defaults to the `<form>` element if no hook is found). And of course this can be overridden when extending the form view or instantiating it (with an option).